### ISSUE TITLE
Build issue triage bot for modules

### ISSUE THREAD

**Issue by gregdek** [ansible/community#4]
Now that we have a new issue template mechanism (thanks Github!) we need to (a) make sure that the template we're using is correct, and (b) make a new bot that works across all three repos to enforce our templating standards.

**Comment by gregdek** [comment]
Depends on #36.

**Comment by gregdek** [comment]
@jctanner did this some time ago. Thanks @jctanner!
